hudi-common/src/main/java/org/apache/hudi/common/util/PartitionPathEncodeUtils.java

```diff
@@ -71,7 +73,7 @@ public static String escapePathName(String path, String defaultPath) {
       if (defaultPath == null) {
         //previously, when path is empty or null and no default path is specified,
```

Comment:
the description should be updated accordingly?

Reply:
done